widget/playground/src/utils/configs.ts

```diff
@@ -0,0 +1,33 @@
+export interface Configs {
+  API_KEY: string;
+}
+
+const RANGO_PUBLIC_API_KEY = 'c6381a79-2817-4602-83bf-6a641a409e32';
+
+let configs: Configs = {
+  API_KEY: RANGO_PUBLIC_API_KEY,
+};
```

Comment:
I think this file changes are pushed by mistake to this PR. :thinking:

Reply:
fixed